docs/user-guide/release-notes/2.11.x.rst

```diff
@@ -2,6 +2,16 @@
 Pulp 2.11 Release Notes
 =======================
 
+EL6 Support Changes
+===================
+
+The 2.11 release is the last release stream with support for EL6. In preparation for 2.12+, EL6
+users are recommended upgrade their Pulp environments to EL7. See the `mailing discussion
```

Comment:
s/upgrade/to upgrade/

I'd also consider s/recommended/encouraged/ , which might sound slightly more natural. It's a matter of taste though.